docs-src/conversations.rst

```diff
@@ -113,9 +114,10 @@ Only public conversations are included by default. You can include DMs and MPIMs
   slack_token = os.environ["SLACK_API_TOKEN"]
   sc = SlackClient(slack_token)
 
+  # Note that `types` is a string
   sc.api_call(
     "conversations.list",
-    types=["public_channel","private_channel"]
+    types="public_channel, private_channel"
```

Comment:
is the space after the comma actually okay? the example on api.slack.com doesn't have a space

Reply:
I don't know if it's preferred without the space, but it works fine with it. @episod ?

Reply:
as long as that works, i'm cool with it!